### PR TITLE
Defining RTCEncodingParameterStats

### PR DESCRIPTION
**Pull request by karthikbr82**

### CODE REVIEW

**Comment by karthikbr82** [comment]
issue #398

File: webrtc-stats.html [hunk 1]
```diff
@@ -4209,6 +4218,331 @@ <h2>
...
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>scalabilityModes</code></dfn> of type <span class=
+                "idlMemberType">DOMString</span>
+              </dt>
+              <dd>
+                <p>Scalability mode identifiers and characteristics are indicated in the table below, which is based
+                on the pre-defined scalability modes in [[AV1]] Section 6.7.5.</p>
+               <table style="border-width:0; width:60%" border="1">
```
Comment (henbos) [inline webrtc-stats.html]: Remove the table in favor of a reference

File: webrtc-stats.html [hunk 2]
```diff
@@ -4209,6 +4218,331 @@ <h2>
           </section>
         </div>
       </section>
+      <section id="encoding-parameters-dict*">
+        <h3>
+          <dfn>RTCEncodingParameterStats</dfn> dictionary
+        </h3>
+        <div>
+          <pre class="idl">dictionary RTCEncodingParameterStats : RTCStats {
```
Comment (henbos) [inline webrtc-stats.html]: DOMString outboundRtp;

File: webrtc-stats.html [hunk 3]
```diff
@@ -4209,6 +4218,331 @@ <h2>
...
+              Dictionary <a class="idlType">RTCEncodingParameterStats</a> Members
+            </h2>
+            <dl data-link-for="RTCEncodingParameterStats" data-dfn-for=
+            "RTCEncodingParameterStats" class="dictionary-members">
+              <dt>
+                <dfn><code>rid</code></dfn> of type <span class="idlMemberType">DOMString</span>
+              </dt>
+              <dd>
+                <p>
+                  Restriction Identifier(rid) of the encoding.
```
Comment (henbos) [inline webrtc-stats.html]: Can we reference the members of [RTCRtpEncodingParameters](https://w3c.github.io/webrtc-pc/#dom-rtcrtpencodingparameters) for all of these, to make it clear where we get this information from?

File: webrtc-stats.html (hunk 2 above)
Comment (henbos) [reply webrtc-stats.html]: OR add encodingParameterId to RTCOutboundRtpStreamStats

Comment (vr000m) [reply webrtc-stats.html]: I favor the second

**Comment by karthikbr82** [comment]
TPAC decision: add "rid" to "outbound-rtp", but don't add "encoding-parameters" (unless we have a specific use case) because this information is already available in getParameters().

Closing this PR.

**Pull request closed without merging**